### PR TITLE
Documents

### PR DESCRIPTION
**Pull request by bwork005**
Changes made in the Readme and Templates

### CODE REVIEW

**Review by enveng-group — CHANGES_REQUESTED** [review]
@bwork005 please go through the files and make changes addressing all my comments from the code review.

File: README.md [hunk 1]
```diff
@@ -1,72 +1,187 @@
...
+Create a new feature or bugfix branch.
+Make your changes, ensuring your commits follow the commit guidelines.
+Submit a pull request to the develop branch for review.
+For more information, refer to our [[CONTRIBUTING.md]CONTRIBUTING.md](CONTRIBUTING.md) file.
+
+---
+
+### Licence
+
+This project is licensed under the [[LICENSE]AGPLv3](LICENSE). Please review the LICENCE file for details on usage and distribution.
```
Comment (enveng-group) [inline README.md:177]: @bwork005 did you check whether this hyperlink works?

File: README.md [hunk 2]
```diff
@@ -1,72 +1,187 @@
...
+
+---
+
+### Contact
+
+For any inquiries or questions regarding this project, feel free to reach out to the team:
+
+Lead Developer: [Your Name](https://github.com/your-username)
```
Comment (enveng-group) [inline README.md:185]: @bwork005 line is incomplete, please revisit.

File: README.md [hunk 3]
```diff
@@ -1,72 +1,187 @@
...
+
+Project Maintainer: [Project Maintainer Name](https://github.com/maintainer-username).
```
Comment (enveng-group) [inline README.md:187]: @bwork005 line is incomplete, please revisit

File: docs/CONTACT.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Contact page
+
+- **[enveng-group](https://github.com/enveng-group)** - enveng-group agallo
+
+- **Email**: [agallo@enveng-group.com.au](mailto:agallo@enveng-group.com.au)
+
+- **GitHub Discussions**: Join the conversation on our [GitHub Discussions] page.
```
Comment (enveng-group) [inline docs/CONTACT.md:7]: @bwork005 did you check whether these links work?

File: docs/CONTRIBUTORS.md
Comment (enveng-group) [inline docs/CONTRIBUTORS.md:1]: @bwork005 where is you?

File: docs/INSTALL.md
Comment (enveng-group) [inline docs/INSTALL.md:1]: @bwork005 delete line 8 to 20

**Review by enveng-group — CHANGES_REQUESTED** [review]
@bwork005 nearly there. Please also ensure you are checking for `markdown-linting` errors, warning and problems in VS Studio Code.

File: docs/CONTACT.md (hunk 4 above)
Comment (bwork005) [reply docs/CONTACT.md:7]: Yes! Changed it now.

File: README.md [hunk 5]
```diff
@@ -1,72 +1,187 @@
...
+
+### Contributing
+
+We welcome contributions to improve the WebApp. If you'd like to contribute, follow these steps:
+
+Fork the repository and clone it to your local machine.
+Create a new feature or bugfix branch.
+Make your changes, ensuring your commits follow the commit guidelines.
+Submit a pull request to the develop branch for review.
+For more information, refer to our [[CONTRIBUTING.md]CONTRIBUTING.md](CONTRIBUTING.md) file.
```
[the remark this answers is not in the document]
Comment (enveng-group) [reply README.md:171]: @bwork005  Please check how line 171 appears when you preview the file. Are you sure we need to repeat `CONTRIBUTING.md` twice?

File: README.md (hunk 1 above)
Comment (enveng-group) [reply README.md:177]: @bwork005 Are you sure the Markdown syntax is correct for line 177 to enclose square brackets within square brackets for the hyperlink?

- See [Markdown Links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links)

File: docs/CONTACT.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# Contact page
+
+- **[enveng-group](https://github.com/enveng-group)** - enveng-group agallo
```
Comment (enveng-group) [inline docs/CONTACT.md:3]: @bwork005 why are you repeating `enveng-group` and why are you using `agallo`?

File: docs/CONTACT.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+
+- **[enveng-group](https://github.com/bwork005)** - enveng-group bwork005
```
Comment (enveng-group) [inline docs/CONTACT.md:5]: @bwork005 Why are you using `enveng-group` as the display name for the link to your Github profile?

Why are you repeating the use `enveng-group` at the end, along with repeating your Github handle in plain text?

Did you preview this page before submitting your commit?

- See [Mentioning People and Teams](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#mentioning-people-and-teams)

**Review by enveng-group — CHANGES_REQUESTED** [review]
@bwork005 Please review all feedback and mark as resolved to track code changes.

- See: [Commenting on Pull Requests] (https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request)

File: .markdown.json
[the remark this answers is not in the document]
Comment (bwork005) [reply .markdown.json:1]: Done!

File: README.md [hunk 8]
```diff
@@ -1,72 +1,186 @@
...
+
+This project is licensed under the [LICENSE](LICENSE). Please review it for details on usage and distribution.
+
+---
+
+### Contact
+
+For any inquiries or questions regarding this project, feel free to reach out to the team:
+
+Lead Developer: [Adrian Agallo](agallo@enveng-group.com.au)
```
Comment (enveng-group) [inline README.md:184]: @bwork005 my surname is incorrect. The correct surname to use is "Gallo".

File: README.md [hunk 9]
```diff
@@ -1,72 +1,186 @@
...
+
+Project Maintainer: [Adrian Agallo](agallo@enveng-group.com.au)
```
Comment (enveng-group) [inline README.md:186]: @bwork005 my surname is incorrect. The correct surname to use is "Gallo".

File: SUPPORT.md [hunk 10]
```diff
@@ -1,29 +1,26 @@
...
 # Support
 
-Thank you for using Obligation Register! If you need help or have any questions, please refer to the following resources:
+Thank you for using Web-App repository! If you need help or have any questions, please refer to the following resources:
 
 ## Documentation
 
-- **User Guide**: Detailed instructions on how to use Obligation Register can be found in the [User Guide](docs/user-guide.md).
-- **API Documentation**: Information about the API endpoints and how to use them can be found in the [API Documentation](docs/api.md).
+- **User Guide**: Detailed instructions on how to use Web-App repo can be found in the CONTRIBUTING.md (develop/CONTRIBUTING.md).
```
Comment (enveng-group) [inline SUPPORT.md:8]: @bwork005 double-check please, whether the link to the CONTRIBUTING file works?

File: SUPPORT.md [hunk 11]
```diff
@@ -1,29 +1,26 @@
...
-## Contributing
 
-We welcome contributions to Obligation Register! If you would like to contribute, please read our [Contributing Guidelines](CONTRIBUTING) for more information on how to get started.
+- **GitHub Discussions**: Join the conversation on our [GitHub Discussions](https://github.com/enssol/web-app/discussions) page.
 
-## Frequently Asked Questions (FAQ)
+## Contributing
 
-- **FAQ**: Common questions and answers can be found in the [FAQ](docs/faq.md) section.
+We welcome contributions to Web-App! If you would like to contribute, please read our [Contributing Guidelines](CONTRIBUTING) for more information on how to get started.
```
Comment (enveng-group) [inline SUPPORT.md:24]: @bwork005 double-check please, whether the link to the CONTRIBUTING file works?

File: docs/CONTACT.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+## Team Members
+
+- **Adrian Agallo** - Project Manager
+- **Adrian Agallo** - Lead Developer
+- **Bhoomi Singh** - Intern
+
+## Support
+
+For support, please our [Support Page](../SUPPORT.md).
```
Comment (enveng-group) [inline docs/CONTACT.md:15]: @bwork005 double-check please, whether the link to the SUPPORT file works?

**Review by enveng-group — CHANGES_REQUESTED** [review]
@bwork005 nearly there. Please double check hyperlinks before resolving.

File: SUPPORT.md (hunk 10 above)
Comment (bwork005) [reply SUPPORT.md:8]: All the links are working.

**Review by enveng-group — APPROVED** [review]

**Pull request merged**